CIP-0098/README.md

```diff
@@ -0,0 +1,474 @@
+---
+CIP: 98?
+Title: DAT Metadata Standard
```

Comment:
```suggestion
Title: Distributed Artifact Token Metadata Standard
```
It's a good name but people finding this PR should more be able to quickly work out what its about without prior knowledge of the acronym.